forecast-admin/forecast/opportunities/models.py

```diff
@@ -160,46 +221,49 @@ class Opportunity(models.Model):
                              validators=[validate_NAICS])
     socioeconomic = models.CharField("Socioeconomic Category", max_length=50,
                                      choices=SOCIOECONOMIC_CHOICES,
-                                     blank=True, null=True)
+                                     default="To Be Determined")
     procurement_method = models.CharField("Procurement Method", max_length=200,
                                           choices=PROCUREMENT_METHOD_CHOICES,
-                                          blank=True, null=True)
+                                          default="To Be Determined")
     contract_type = models.CharField("Contract Type", max_length=200,
                                      choices=CONTRACT_TYPE_CHOICES,
-                                     blank=True, null=True)
+                                     default="To Be Determined")
     competition_strategy = models.CharField(max_length=200,
                                             choices=COMPETITION_CHOICES,
-                                            blank=True, null=True)
-    price_min = models.CharField(max_length=200, blank=True, null=True)
-    price_max = models.CharField(max_length=200, blank=True, null=True)
+                                            default="To Be Determined")
+    price_min = models.DecimalField(max_length=200, decimal_places=2,
+                                    max_digits=16, blank=True, null=True)
+    price_max = models.DecimalField(max_length=200, decimal_places=2,
+                                    max_digits=16, blank=True, null=True)
     delivery_order_value = models.CharField(max_length=200,
                                             blank=True, null=True)
     incumbent_name = models.CharField("Incumbent Contractor Name",
                                       max_length=200, blank=True, null=True)
     new_requirement = models.CharField(max_length=200,
                                        choices=NEW_REQUIREMENT_CHOICES,
-                                       blank=True, null=True)
+                                       default="To Be Determined")
     funding_agency = models.CharField(max_length=200,
                                       choices=FUNDING_AGENCY_CHOICES,
-                                      blank=True, null=True)
+                                      default="To Be Determined")
     estimated_solicitation_date = models.DateField(blank=True, null=True)
     fedbizopps_link = models.CharField(max_length=200, blank=True, null=True)
-    estimated_fiscal_year = models.CharField(max_length=200,
-                                             blank=True, null=True)
-    estimated_fiscal_year_quarter = models.CharField(max_length=200,
-                                                     blank=True, null=True)
+    # TODO: going to make these choices...
+    estimated_fiscal_year = models.IntegerField(default=2016)
```

Comment:
Related. When you do unit tests, do you test for things like model fields? I mean, I *could* write a function that tests whether it fails to create an Opportunity if I used `2016a`, but I've been assuming that Django will handle that for me. :grimacing: